contrib/linux/sensors/file_watch_sensor.py

```diff
@@ -44,8 +44,8 @@ def cleanup(self):
 
             try:
                 self._tail.notifier.stop()
-            except Exception:
-                pass
+            except Exception as e:
+                print(f"Error: tail notifier is still running. {e}")
```

Comment:
Can you do a `self._logger.exception("Unable to stop the tail notifier.")` instead of print? This should include the exception by default.